### DISCUSSION TITLE
getContext() returns undefined when accessing another block's context

### DISCUSSION THREAD

**Discussion by jamesina2** [WordPress/gutenberg#59765] [Q&A]
Great job on Interactivity API so far! 

I'm trying a proof of concept where I can access another block's context. I initialize two blocks with:

`npx @wordpress/create-block@latest block-1 --template @wordpress/create-block-interactive-template`
`npx @wordpress/create-block@latest block-2 --template @wordpress/create-block-interactive-template`

I give their stores different names and try basic functions for each, everything is working fine there. 

However, when I try to access block-1 context from block-2 eg. `const anotherBlockContext = getContext('block-1')` I get always  `undefined`

My block-2 view.js example:

```
store( 'block-2', {
	actions: {
		toggle: () => {
			const context = getContext();
			console.log(context); // works 
			const anotherContext = getContext('block-1');
			console.log(anotherContext); // undefined
			context.isOpen = ! context.isOpen;
		},
	}
} );
```

Any particular reason why this would return `undefined`?

**Comment by michalczaplinski** [comment]
How did you define the context of your block? Could you share the `render.php` of your block?
